### PR TITLE
nav bar sticks to top of page

### LINKED ISSUE #31: Fix Navigation bar
# Summary
Currently, when users navigate through our website, they have to scroll all the way back to the top to switch pages. By making the navigation bar collapsible as they scroll down, we can improve the overall user experience

###  Currently:

Image

Image

### Idea:
Image

### PR DESCRIPTION
**Pull request by jaidaduson**
ScreenRecording2024-11-11at5 43 20PM-ezgif com-optimize
close #31

### CODE REVIEW

**Review by boldenj2 — CHANGES_REQUESTED** [review]
Can you make the buttons smaller or the div surrounding the buttons bigger.

image

File: src/components/headerTransparent.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../styles/headerTransparent.css'; 
+import StickyNavBar from './stickyNavBar';
+
+const HeaderTransparent = () => {
+  return (
+    <div className="sticky-header">
+      <StickyNavBar/>
```
Comment (scuruchima1) [inline src/components/headerTransparent.js]: I am a little bit confused here. Can we combine this component with the StickyNavBar component? I don't see other functionality here besides the div and class, unless I'm missing something.

**Review by scuruchima1 — COMMENTED** [review]
Overall functionality looks good. Left a comment about the possibility of combining components. Also +1 to Bolden's comments.

File: src/components/headerTransparent.js (hunk 1 above)
Comment (alognz) [reply src/components/headerTransparent.js]: you're right, forgot we werent using this component anymore but have deleted it now mb

**Comment by alognz** [comment]
i made the buttons smaller & removed redundant files with 2 latest commits. lmk if it looks good now.

**Review by boldenj2 — APPROVED** [review]
lgtm

**Review by scuruchima1 — APPROVED** [review]
LGTM

**Pull request merged**